main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+)
+
+func CreatePhoneBook(names []string, numbers []string) map[string]string {
+	x := len(names)
+	phonebook := make(map[string]string)
+	for ; x > 0; x-- {
+		phonebook[names[x-1]] = numbers[x-1]
+	}
+	return phonebook
+}
+
+func main() {
+	var names, numbers []string
+	f, err := os.Open("phone_numbers.txt")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+	s := bufio.NewScanner(f)
+	for tmp := 1; s.Scan(); tmp++ {
+		input := s.Text()
+		if tmp%2 != 0 {
+			names = append(names, input)
+		} else {
+			numbers = append(numbers, input)
+		}
+	}
```

Comment:
nit: This can be replaced with 

```
for s.Scan() {
        names = append(names, s.Text())
        s.Scan()
        numbers = append(numbers, s.Text())
}
```